community/cypher/cypher-compiler-3.3/src/main/scala/org/neo4j/cypher/internal/compiler/v3_3/PlannerName.scala

```diff
@@ -59,12 +59,13 @@ object PlannerNameFor {
 
   def apply(name: String): PlannerName = name.toUpperCase match {
     case IDPPlannerName.name => IDPPlannerName
-    case DPPlannerName.name => DPPlannerName
-    case "COST" => CostBasedPlannerName.default
-    case "DEFAULT" => CostBasedPlannerName.default
+    case DPPlannerName.name  => DPPlannerName
+    case "COST"              => CostBasedPlannerName.default
```

Comment:
Aha, I actually do like aligned arrows and some operators like assignment. Not passionately though.